### ISSUE TITLE
QTree looks "wonky" on High DPI monitors

### ISSUE THREAD

**Issue by rfox12** [quasarframework/quasar#4836]
I develop on two monitors... on my "regular" DPI monitor QTree looks just fine.  On my high DPI monitor you can see jaggy edges everywhere.  See attached screenshots (these are directly from the QTree doc page).  I think the obvious solution is that the community should buy all quasar devs new high DPI monitors. :)  

Not a high priority for me. Just putting the issue here for tracking.

![screen1](https://user-images.githubusercontent.com/33066940/62658190-316b5100-b936-11e9-8d65-5d4556273503.png)
![screen2](https://user-images.githubusercontent.com/33066940/62658192-33351480-b936-11e9-88ac-655b9e180dd8.png)

**Comment by hawkeye64** [comment]
```
I think the obvious solution is that the community should buy all quasar devs new high DPI monitors. :)
```
Works for me. 😂

**Comment by rstoenescu** [comment]
Fix will be available in "quasar" v1.5.10.
